app/celery/scheduled_tasks.py

```diff
@@ -189,10 +191,18 @@ def delete_invitations():
 @notify_celery.task(name='timeout-sending-notifications')
 @statsd(namespace="tasks")
 def timeout_notifications():
-    updated = dao_timeout_notifications(current_app.config.get('SENDING_NOTIFICATIONS_TIMEOUT_PERIOD'))
-    if updated:
+    notifications = dao_timeout_notifications(current_app.config.get('SENDING_NOTIFICATIONS_TIMEOUT_PERIOD'))
+
+    if notifications:
+        for notification in notifications:
+            # queue callback task only if the service_callback_api exists
+            service_callback_api = get_service_callback_api_for_service(service_id=notification.service_id)
+
+            if service_callback_api:
```

Comment:
Is this normal operation i.e. if a service hasn't got it configured then service_callback_api is None? Or is this defensive as in something failed?

If the latter add a log statement etc

Reply:
Yes. That's correct. We don't want to put a task on the queue if they haven't got it configure.